src/liblx/xml/common/common.h

```diff
@@ -70,9 +70,9 @@
  * into the XML produced by libSBML.  This is used by
  * StringBuffer_appendReal.
  */
-#define LIBSBML_FLOAT_FORMAT "%.15g"
+#define LIBLX_FLOAT_FORMAT "%.15g"
 
-#define LIBSBML_DOUBLE_PRECISION 15
+#define LIBLX_DOUBLE_PRECISION 15
 
 static const int SBML_INT_MAX = 2147483647;
```

Comment:
Do we still want these listed as `SBML_` ?

Reply:
i could go with either definition, LIBLX is more consistent